### DISCUSSION TITLE
[ Channel Configuration ] View relation when config channels are linked to system groups

### DISCUSSION THREAD

**Discussion by maceoparkore** [uyuni-project/uyuni#6935] [Ideas]
Hi,

i think it will be useful if we could see the relations between sls  config channel and systems groups from the config channel side.

Regards.

**Comment by admd** [comment]
@maceoparkore unless I misunderstood your idea, one can already see the this relation from any config channel 

channel -> Systems-> subscribed systems. 

Let me know if you meant something different.

**Comment by admd** [comment]
never mind, I see you meant system groups and not individual systems. It makes sense to me and I see it being useful. We could look into it in future.

**Reply by maceoparkore** [reply]
Hello admd, 

yes that's the point. for end user to view from configuration channel the groups relationships as far as org mapping. We manage all our configuration channels by mapping them to system groups and could always have the possibilities to manage custom systems by org-group-system hierarchy.

Regards,
